src/variables/variables.ts

```diff
@@ -23,11 +23,14 @@ import {
     IFees,
     httpStatusCode,
 } from 'src';
+import {epsBucketUrls, epsUrls} from 'src/variables/constants';
 
 export const environment: Required<IEnvironment> = {
     type: environmentTypes.production,
     url: environmentUrls.production,
-    path: environmentPath
+    path: environmentPath,
+    epsUrl: epsUrls.production,
+    epsBucketUrl: epsBucketUrls.production,
```

Comment:
These need to be optional for now, a lot of places use the environment and if we make those values required this will cause typescript errors on someplaces.

Also, is there a PR change on Checkout to send those 2 values as part of the environment added to the window?